### PR TITLE
Gen key

### PR DESCRIPTION
**Pull request by CodeSandwich**
CLI tooling for public and private key generation

### CODE REVIEW

File: src/settings/command_arguments.rs [hunk 1]
```diff
@@ -96,6 +97,24 @@ pub struct InitArguments {
...
 }
 
+#[derive(StructOpt, Debug)]
+pub struct GeneratePrivKeyArguments {
+    /// Type of a private key
+    #[structopt(long = "type")]
+    pub key_type: GenPrivKeyType,
+}
+
+arg_enum! {
```
Comment (NicolasDP) [inline src/settings/command_arguments.rs]: neat. didn't see that last time <3

**Comment by NicolasDP** [comment]
what about removing the old command then ? It is deprecated and not really usable.

also please, can you update the README to provide the necessary information regarding how to generate the keys ?

**Comment by CodeSandwich** [comment]
@NicolasDP I'm not sure if we should already remove the old command and update README. The old key generator produces hex-encoded keys while the new one bech32-encoded. Genesis and xprv files all work on hex-encoded keys, so until we accept them in bech32 the new key generator is pretty much useless.

**Comment by vincenthz** [comment]
it's ok to do another PR

**Pull request merged**